### PR TITLE
deps: Bump minidump (0.22.0 to 0.23.0) in script/

### PR DESCRIPTION
**Pull request by DeeDeeG**
### Identify the Bug

Fixes: https://github.com/atom-community/atom/issues/363

### Description of the Change

Bump minidump from 0.22.0 to 0.23.0 in script/package.json

### Alternate Designs

None

### Possible Drawbacks

None expected.

### Verification Process

https://github.com/atom-community/atom/issues/363 seems to be fixed by updating `minidump` to the latest version, in `script/package.json`. 0.22.0 --> 0.23.0

<details>

I tested this in a fresh install of the latest [Manjaro Linux](https://manjaro.org/), which is a type of Arch Linux. I got the same error situation as @DAC324 reported.

I tried debugging it. `dump_syms` had blank output on stdout and stderr, so that is why there was no detailed or helpful error message being printed. The binary being spawned really had no output. I think it's fixed by this upstream commit of breakpad? ["avoid dump_syms crashing if selected arch is not found"](https://chromium.googlesource.com/breakpad/breakpad/+/c85eb4a59b618f3beaad5445ceb1f865ffa8efdf)... Maybe.

Something about the way `dump_syms` exited made it hard to catch and handle properly, I guess. So it "leaked through" and halted the build script, even though we're supposed to be handling (ignoring) the error. 

In any case, it works for me now with updated `minidump`.

</details>

### Release Notes

deps: Bump minidump from 0.22.0 to 0.23.0 in script/package.json

### CODE REVIEW

**Comment by DeeDeeG** [comment]
Note: **Please wait for CI to pass** for this PR -- I don't have access to a good Windows machine and macOS machine to build on, so I want to make sure CI has no issues on those OSes. Also, CI will test on Ubuntu, whereas I have only tested this on Arch Linux so far.

**Review by icecream17 — APPROVED** [review]

**Comment by aminya** [comment]
@icecream17 CI was not triggered for this one. You should make sure that the CI passes before merging.

**Comment by aminya** [comment]
I started a manual run on the master branch. Not sure why it is not triggered.

**Comment by DeeDeeG** [comment]
I was trying to get CI to run manually, had a slightly hard time logging in, etc.

I'll see how the CI run of `master` goes.

**Comment by DeeDeeG** [comment]
It has issues on Windows, so I think it should be reverted. :/

See: https://github.com/electron/node-minidump/issues/51

Our CI failure matches what others are seeing on Windows in that issue.

**Comment by DeeDeeG** [comment]
Does anyone mind if I push a revert commit directly to `master`, or should I do it as a PR?

**Pull request merged**
